README.md

```diff
@@ -69,11 +118,19 @@ metadata:
 spec:
   module: registry://ghcr.io/kubewarden/policies/cel-policy:latest
   settings:
+    failurePolicy: Fail # this settings is optional. When not defined, the default value is `Fail`
+    paramKind:
+      apiVersion: v1
+      kind: ConfigMap
+    paramRef:
+      name: "my-params"
+      namespace: "default"
+      parameterNotFoundAction: Deny
     variables:
       - name: "replicas"
         expression: "object.spec.replicas"
     validations:
-      - expression: "variables.replicas <= 5"
+      - expression: "variables.replicas <= params.data.maxreplicas"
```

Comment:
I would include also the definition for the ConfigMap in this same YAML block to tie everything together. That way people don't need to know about VAPs.

Reply:
Sorry, I did not understand the request. Do you want to add a comment with an example of the params ConfigMap resource in the yaml block?